WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2284,3 +2284,23 @@ WhiteBlackListViewRowCell                  = font-weight:normal;
 WhiteBlackListRemoveItemCell               = width:@BigIconWidth@; @ListItemIconWrapper@
 WhiteBlackListViewItemsNumberUsed          = margin-top:@WhiteBlackViewInnerSpacing@;
 
+######################################
+#   OptionsItems ListView
+######################################
+
+OptionsItemsListContainer                  = @PanelBorder@
+OptionsItemsListViewItemsInnerSpacing      = 10px;
+OptionsItemsListColumnHeaderWrapper        = border-left:0px;
+OptionsItemsListColumnHeaderCell           = padding-left:@OptionsItemsListViewItemsInnerSpacing@;
+OptionsItemsListRow                        = overflow:hidden;
+OptionsItemsListLastRow                    = border-bottom:none;
+OptionsItemsListRows                       = overflow:visible;
+OptionsItemsListRowCell                    = padding-left:@OptionsItemsListViewItemsInnerSpacing@; text-align:left;
+OptionsItemsListRowCellSelected            = line-height:1.7rem;
```

Comment:
why do we need to change line height when selected? Shouldn't this be same irrespective of selected/not selected?